### ISSUE TITLE
4.0.4 ISO fails to boot on HP Z8 Workstation

### ISSUE THREAD

**Issue by permezel** [QubesOS/qubes-issues#6448]
**Qubes OS version**

4.0.4

**Affected component(s) or functionality**

Boot/Install ISO

**Brief summary**

Tried booting from both 64Gig SanDisk USB drive and 1T SanDisk SSD drive.  Both times got as far as relinquishing to console to Xen and then nothing.

**How Reproducible**

100%

**To Reproduce**

1. Download ISO
2. Verify sig with GPG
3. dd to USB Drive
4. Insert in HP Z8 Workstation USB slot
5. boot (with liberal application of fresh electrons)
6. wait for install

**Expected behavior**

In the past, I have been able to get into the install screen, and install onto this machine.

**Actual behavior**

Just a blank screen.

**Screenshots**

![image](https://user-images.githubusercontent.com/2313876/110456050-d9e62600-8114-11eb-8074-de66903a08df.png)

Once it gets here, it clears the screen and hangs.  No reboot.  No response to any key sequences on the non-USB keyboard.  Not even CTL-ALT-DEL.  Need to hold the power button down for n-seconds to get it to power off.

**Additional context**

It did used to boot/install/run on this machine, but is currently not running on it.
Just recently installed fedora33 workstation on it with no problems.

**Solutions you've tried**

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

**Comment by permezel** [comment]
Just re-verified the SSD contents:

```
bash-3.2# dd if=/dev/disk2 bs=1m count=4959 | gpg --verify ~dap/Downloads/Qubes-R4.0.4-x86_64.iso.asc -
gpg: Signature made Fri  5 Mar 00:09:55 2021 AEST
gpg:                using RSA key 5817A43B283DE5A9181A522E1848792F9E2795E9
gpg: key 1848792F9E2795E9: public key "Qubes OS Release 4 Signing Key" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: Good signature from "Qubes OS Release 4 Signing Key" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 5817 A43B 283D E5A9 181A  522E 1848 792F 9E27 95E9
bash-3.2# :              5817 A43B 283D E5A9 181A  522E 1848 792F 9E27 95E9 -- チェクします

```

**Comment by andrewdavidwong** [comment]
Can you please confirm that this hardware satisfies all the system requirements and that you have gone through the installation troubleshooting documentation?

**Comment by permezel** [comment]
Per https://www.qubes-os.org/hcl/ this platform is supported.  I used to run 4.0 on it.

Perhaps it is due to an EFI boot issue.  I had disabled legacy boot option, and when I enabled it, even though it used EFI to boot, it now comes up to the install screen.

Something to do with the option ROMs being executed?

Anyway, I currently do not have spare disk on this system to proceed with the install, so I just got as far as disk selection and then quit.

BTW: I tried iSCSI, but that died, not that I really care for iSCSI.
